contracts/test/test-helpers/helpers.ts

```diff
@@ -0,0 +1,224 @@
+import { BigNumber, Contract, ContractTransaction } from "ethers";
+import type { providers } from "ethers";
+import { assert } from "chai";
+import { ethers } from "hardhat";
+import cbor from "cbor";
+
+export const constants = {
+  ZERO_ADDRESS: "0x0000000000000000000000000000000000000000",
```

Comment:
I think Ethers offers these too, and we use ours in some places and ethers in another place. Would be nice to consolidate.

Reply:
Removed in favour of ethers constants